arch/arm/core/cortex_m/Kconfig

```diff
@@ -285,6 +285,12 @@ config SW_VECTOR_RELAY
 	  vector table relocation mechanisms or for Cortex-M0+
 	  (or an Armv8-M baseline core) with no VTOR and no other hardware
 	  relocation table mechanisms.
+
+config PLATFORM_SPECIFIC_INIT
+	bool
+	prompt "Enable platform (SOC) specific startup hook"
+	default n
```

Comment:
Some help text, here, would be very nice, since this will be a generic Cortex-M K-option (even though the option is self-descriptive)

Reply:
Added a simple help text.